FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +22,60 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let _ = self else {
+				return
+			}
+
+			switch result {
+			case .success((let data, let response)):
+				completion(FeedItemMapper.map(data, from: response))
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+final class FeedItemMapper {
+	private static var OK_200: Int {
+		return 200
+	}
+
+	static func map(_ data: Data, from response: HTTPURLResponse) -> FeedLoader.Result {
+		guard response.statusCode == self.OK_200 else {
```

Comment:
Explicit `self` is not used in this codebase. Please remove `self` to make code consistent with the rest of the codebase.

```suggestion
		guard response.statusCode == OK_200 else {
```

Reply:
Updated.